### PR TITLE
fix: improve arraybuffer example

### PR DESCRIPTION
**Pull request by hubigitt**
#### Summary

- Change var to const where possible
- Change naming

#### Motivation

To refresh code examples.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/arraybuffer/arraybuffer/index.md [hunk 1]
```diff
@@ -19,7 +20,7 @@ The **`ArrayBuffer()`** constructor is used to create
 ## Syntax
 
 ```js
-new ArrayBuffer(length)
+new ArrayBuffer(length);
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/arraybuffer/arraybuffer/index.md:23]: We don't use semicolons in the "Syntax" section.

**Review by wbamberg — APPROVED** [review]
Thanks!

This is great except that we don't use semicolons in "Syntax".

**Review by wbamberg — CHANGES_REQUESTED** [review]
Ooops sorry, meant to request changes!

**Comment by wbamberg** [comment]
@teoli2003 points out that in our meta-docs we do have semicolons in "Syntax": https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections . But when @Elchi3 revised them all last year this came up and we decided no semicolons (https://github.com/mdn/content/pull/4569#pullrequestreview-647692811) - and this is our general practice in the JS docs AFAIK (e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice#syntax).

So should we update the meta-docs, or update all the other pages?

I think I am mildly anti-semicolon, as these are not actual JS syntax. But syntax sections are kind of uncanny valley territory, where they are almost JS but are not quite, and this is always going to be a source of confusion (and will get worse with Prettier/ESLint).

**Comment by hubigitt** [comment]
Ooops, sorry! Already did the changes.

**Review by wbamberg — APPROVED** [review]
Let's merge this anyway and update it later if we decide to.

Thanks @hubigitt !

**Pull request merged**